### PR TITLE
feat: implement forum helpers

### PR DESCRIPTION
**Pull request by sillyElvy**
## Checklist

- [x] The ``pre-commit`` code linter has been run over all edited files to ensure the code is linted.
- [ ] I've ensured the change(s) work on `3.8.6` and higher.

I've made this pull request: (check all that apply)
  - [x] For the documentation
  - [x] To add a new feature
  - [x] As a general enhancement
  - [ ] As a refactor of the library/the library's code
  - [ ] To fix an existing bug
  - [ ] To resolve #ISSUENUMBER

This is:
  - [ ] A breaking change

### CODE REVIEW

**Comment by sillyElvy** [comment]
shit forgot docs

**Comment by sillyElvy** [comment]
done.

File: interactions/api/error.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def lookup(code: int) -> str:
             11: "Error creating your command.",
             12: "Invalid set of arguments specified.",
             13: "No HTTPClient set!",
+            14: "Fatal conflict between object and attempted action (cannot run this action on current object).",
```
Comment (Toricane) [inline interactions/api/error.py]: I don't think having both of those descriptions is good, maybe put one of them?

**Review by Toricane — CHANGES_REQUESTED** [review]
A suggestion

**Review by Toricane — APPROVED** [review]
LGTM!

**Pull request merged**